packages/stark-ui/src/modules/pretty-print/components/pretty-print.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="stark-pretty-print">
+	<div *ngIf="highlightingEnabled">
+		<div [innerHTML]="prettyString"></div>
```

Comment:
Is it secure to bind to the innerHTML property like that?
I mean, what happen if the prettyString is an HTML and it contains for example an <script> tag ?